bqschema_suite_test.go

```diff
@@ -1,10 +1,10 @@
-package bqschema_test
+package bqschema
```

Comment:
All of what you say is true. I made this change because the usage of package `*_test` for tests has two specific purposes: avoiding an import cycle, and  black box testing. Neither of those is necessary here.

If there were a true black box test, I agree it should be in either `package bqschema_test` or another package.